cli/cmdargs/parser.go

```diff
@@ -76,6 +79,7 @@ const (
    Examples:
     * 'int:42' is an integer with a value of 42
     * '42' is an integer with a value of 42
+	* 'nil' is a parameter with Any type and nil value (corresponds to Null stackitem)
```

Comment:
Indentation seems to still be wrong here.

Reply:
Sorry, I've fixed indentation at another place. Fixed.